packages/orbit-components/src/Tag/index.tsx

```diff
@@ -168,6 +173,25 @@ StyledClose.defaultProps = {
   theme: defaultTheme,
 };
 
+const StyledIconContainer = styled.div`
```

Comment:
I think you can use Stack here. Just wrap `iconLeft`, `children` and `CloseContainer` inside Stach with props `inline`, `align="center"` and `spacing="XSmall` :)

Reply:
Yeah, I would personally prefer to have less code. Additional div won't make any difference in the markup, but the custom styled-component will be bundled, while Stack will be tree just reused, so less addition to bundle ;)

Reply:
Fair point. I'll leave it like this for now but maybe we can reassess all this during component refactoring for new styling solution and have a consistent approach across all components in Orbit 🙂